vcd/resource_vcd_service_account.go

```diff
@@ -0,0 +1,345 @@
+package vcd
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"net/url"
+	"strings"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/vmware/go-vcloud-director/v2/govcd"
+	"github.com/vmware/go-vcloud-director/v2/types/v56"
+)
+
+func resourceVcdServiceAccount() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: resourceVcdServiceAccountCreate,
+		UpdateContext: resourceVcdServiceAccountUpdate,
+		ReadContext:   resourceVcdServiceAccountRead,
+		DeleteContext: resourceVcdServiceAccountDelete,
+		Importer: &schema.ResourceImporter{
+			StateContext: resourceVcdServiceAccountImport,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:        schema.TypeString,
+				Required:    true,
+				ForceNew:    true,
+				Description: "Name of service account",
+			},
+			"org": {
+				Type:     schema.TypeString,
+				Optional: true,
+				ForceNew: true,
+				Description: "The name of organization to use, optional if defined at provider " +
+					"level. Useful when connected as sysadmin working across different organizations",
+			},
+			"software_id": {
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: "UUID of software, e.g: 12345678-1234-5678-90ab-1234567890ab",
+			},
+			"role": {
```

Comment:
This property should be called `role_id`

Reply:
Updated.